.travis.yml

```diff
@@ -3,4 +3,4 @@ script: "./script/cibuild"
 gemfile: "this/does/not/exist"
 env: GITHUB_API_TOKEN=36a1b2a815b98d755528fa6e09b845965fe1e046 # token of user: some-public-token without any permissions to avoid api rate limit
 rvm:
-  - "1.8.7"
+  - "2.1.5"
```

Comment:
>  to look like the one that puppet-template uses

I assume you mean having the `rvm` block look like this?
```
rvm:
  - "1.8.7"
  - "2.0.0"
```
One of the dependencies `highline` requires Ruby version >= 1.9.3 so `1.8.7` isn't going to work. Will having just `2.0.0` suffice?